build.gradle

```diff
@@ -94,7 +94,7 @@ subprojects { project ->
         key = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_API_KEY')
         configurations = ['archives']
         pkg {
-            repo = 'maven'
+            repo = 'arrow-kt'
```

Comment:
Will this affect current releases?

Reply:
Nope, the repo name in Bintray is `arrow-kt` 👀 https://bintray.com/arrow-kt/arrow-kt
In any case, this is only used when running the `bintrayUpload` task which is now disabled https://github.com/arrow-kt/arrow/blob/9e4ced86a11fa167b9fb548a51e2e25f769bcea1/deploy-scripts/deploy_release.sh#L19